pkg/services/queryparser/query_parser.go

```diff
@@ -74,6 +76,7 @@ var _ QueryParser = &queryParser{}
 // EQ           -> VALUE | QUOTED_VALUE
 // NOT_EQ       -> VALUE | QUOTED_VALUE
 // LIKE         -> VALUE | QUOTED_VALUE
+// ILIKE        -> VALUE | QUOTED_VALUE
```

Comment:
Please update https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pull/1203/files#diff-b9f3832eada41015efaf6287638235412d9395abab3f46d884919979457dcdebR75 too to:
```
// COLUMN       -> EQ | NOT_EQ | LIKE | ILIKE
```

Reply:
Good catch. Fixed.